sound/soc/sof/ipc4-pcm.c

```diff
@@ -122,8 +124,9 @@ static int sof_ipc4_trigger_pipelines(struct snd_soc_component *component,
 	/* update PAUSED state for all pipelines that were just triggered */
 	for (i = 0; i < data->count; i++) {
 		for (j = 0; j < pipeline_list->count; j++) {
-			pipe_widget = pipeline_list->pipe_widgets[j];
-			pipeline = pipe_widget->private;
+			spipe = pipeline_list->pipelines[j];
+			pipe_widget = spipe->pipe_widget;
```

Comment:
Is this correct?
The `pipeline`, which we are going to change the state at line 129/132 is the `pipeline_list->pipe_widgets[0]`'s always.

Reply:
oh i removed the 			pipeline = pipe_widget->private;
by mistake.

Reply:
fixed now